static/src/javascripts/boot.js

```diff
@@ -58,13 +57,6 @@ const go = () => {
 
             if (!recordedConsentTime) {
                 recordedConsentTime = true;
```

Comment:
not a blocker, but I do think the containing code block can also be removed, as `recordedConsentTime` is not used anywhere else.

Reply:
That makes sense. I am not sure if the comment is up to date or not cause I am trying to find `App.tsx` in Frontend but can't find it!

I understand that I will remove the variable `recordedConsentTime` and the conditional statement that checks it. Please correct me if I missed something.

Reply:
I think it should have said `App.tsx` in `dotcom-rendering`